src/utils/error.ts

```diff
@@ -1145,3 +1145,45 @@ const TOPIC_MGT_SERVER_TO_CLIENT_CODE: ServerToClientCode = {
   INTERNAL: 'INTERNAL_ERROR',
   UNKNOWN: 'UNKNOWN_ERROR',
 };
+
+/**
+ * Firebase Phone Number Verification error code structure. This extends PrefixedFirebaseError.
+ */
+export class FirebasePnvError extends PrefixedFirebaseError {
+  /**
+     * @param info - The error code info.
+     * @param message - The error message. This will override the default message if provided.
+     * @constructor
+     * @internal
+     */
+  constructor(info: ErrorInfo, message?: string) {
+    // Override default message if custom message provided.
+    super('fpnv', info.code, message || info.message);
+
+    /* tslint:disable:max-line-length */
+    // Set the prototype explicitly. See the following link for more details:
+    // https://github.com/Microsoft/TypeScript/wiki/Breaking-Changes#extending-built-ins-like-error-array-and-map-may-no-longer-work
+    /* tslint:enable:max-line-length */
+    (this as any).__proto__ = FirebasePnvError.prototype;
+  }
+}
+
+export class FpnvErrorCode {
```

Comment:
Please use a type here to keep things consistent with the rest of the codebase. We have some work planned for error handling revamp so having it consistent here will reduce conflicts. Thanks!

For example see: https://github.com/firebase/firebase-admin-node/blob/179dab70b7ccc89185fc90e5c7a48ffecef60102/src/app-check/app-check-api-client-internal.ts#L228

https://github.com/firebase/firebase-admin-node/blob/179dab70b7ccc89185fc90e5c7a48ffecef60102/src/data-connect/data-connect-api-client-internal.ts#L657